rest/handler.go

```diff
@@ -405,6 +407,10 @@ func (h *handler) validateAndWriteHeaders(method handlerMethod, accessPermission
 	if h.privs != adminPrivs {
 		var err error
 		if err = h.checkAuth(dbContext); err != nil {
+			// if auth fails we still need to record bytes read over the rest api for the stat, to do this we need to read
+			// the body to populate the bytes count on the CountedRequestReader struct
+			_, _ = h.readBody()
```

Comment:
Don't think `h.readBody()` is required after the last commit/update - as `GetBodyBytesCount()` will read if required.